CMakeLists.txt

```diff
@@ -105,6 +109,18 @@ else()
   set(CMAKE_CXX_STANDARD 11)
 endif()
 
+string(FIND "${LLVM_CONFIG_CXXFLAGS}" "-stdlib=libc++" LLVM_LIBCXX)
```

Comment:
Will this work for non-Clang environments?

Reply:
No. But it wouldn't link anyway. If you have LLVM built with libc++ you need to build Halide with it also.